src/engine/Settings.cs

```diff
@@ -141,6 +141,8 @@ private Settings()
 
     public int CloudSimulationHeight => Constants.CLOUD_Y_EXTENT / CloudResolution;
 
+    public string SelectedLanguage { get; set; } = "en";
```

Comment:
This option should reapply when the game is restarted, like the graphics options.
Also the default option should be null, which means "whatever godot detects as the user's current language"